app/components/web-components/vl-table-actions/template.hbs

```diff
@@ -44,6 +44,7 @@
     target="_blank"
     bubbles=false
     class="vl-button vl-button--link vl-button--icon"
+    data-test-link-to-subcase-overview=""
```

Comment:
Niet echt een opmerking, maar gewoon om mee te geven, als je data-test-tags zet als eerste parameter, dan lijkt het systeem slim genoeg om te zien dat het een data-tag is en hoef je geen lege value mee te geven. In een later stadium kunnen we ook beginnen met de tags te vervangen waar ik in het begin =true heb gezet.
bv:
```
  {{#link-to
    data-test-link-to-subcase-overview
    "cases.case.subcases.subcase.overview"
    (await row.subcase.case.id)
    (await row.subcase.id)
    target="_blank"
    bubbles=false
    class="vl-button vl-button--link vl-button--icon"
  }}
```

Reply:
Eens je de angle-brackets notatie gebruikt, mag je ze eender waar zetten zonder value geloof ik ...